### DISCUSSION TITLE
latex 参考文献引用 专利 问题

### DISCUSSION THREAD

**Discussion by Ethan1239** [note286/xduts#165] [Q&A] [answered]
论文要求 专利引用类型为[P]； 但是使用以下代码 输出类型不对。
@manual{胡旭2023,
      author = {胡旭 and 杜涛涛 and 王萌},
      title = {基于FPGA平台的多路并行DDS双向线性扫频方法、系统及介质},
      edition = {CN114115436B},
      year = {2023},
      pages = {12},
      publisher={Patent}
}  
胡旭, 杜涛涛, 王萌. 基于 FPGA 平台的多路并行 DDS 双向线性扫频方法、系统及介质[A]. CN114115436B. Patent, 2023: 12.

**Comment by sikouhjw** [comment, marked as answer]
有没有可能专利用 `@patent`

**Reply by Ethan1239** [reply]
感谢感谢。但是 请问生成的文献中 为什么会多个冒号？？？
胡旭, 杜涛涛, 王萌. 基于 FPGA 平台的多路并行 DDS 双向线性扫频方法、系统及介质 **:** [P].2023.
代码如下：
@patent{胡旭2023,
      author = {胡旭 and 杜涛涛 and 王萌},
      title = {基于FPGA平台的多路并行DDS双向线性扫频方法、系统及介质},
      edition = {CN114115436B},
      year = {2023},
      pages = {12}
}

**Reply by sikouhjw** [reply]
你没发现你的 `edition` 没输出吗？自己对比一下
```tex
@patent{zhangkaijun2012guidao,
  author       = {张凯军},
  title        = {轨道火车及高速轨道火车紧急安全制动辅助装置},
  year         = {2012},
  date         = {2012-04-05},
  number       = {201220158825.2},
}
```

**Reply by Ethan1239** [reply]
感谢感谢，问题已解决。
